docs/openapi.json

```diff
@@ -254,6 +254,9 @@
           "400": {
             "description": "Request failed. The incoming data specification seems malformed."
           },
+          "404": {
```

Comment:
Hmm, 404 might be better than 500, but it may also be misleading, in the usual sense that it indicates that the resource is not there, and please retry later... But the endpoint is actually there, only the wanted values are not in agreement with what the instance supports.

Two possibilities: 

- Perhaps we can find another status code to use (or misuse), such as `406 Not Acceptable` or something? Anyway, since this is for internal REANA component communication, not seen by the client, we are somehow "freer" in the choice of exit status codes.

- Perhaps we catch this situation sooner, and we won't allow execution of such workflow at all in the first place? In this way there is no need to change anything here.  (Note: this is linked to the `info` and to the dynamical checking idea https://github.com/reanahub/reana-client/issues/567#issuecomment-933560897. We'd refuse to accept such workflows already at the `r-server` early stages.)